ledger/acctdeltas_test.go

```diff
@@ -1067,6 +1067,135 @@ func BenchmarkLookupKeyByPrefix(b *testing.B) {
 	}
 }
 
+func TestKVStoreNilBlobConversion(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	const inMem = false
+
+	log := logging.TestingLog(t)
+	log.SetLevel(logging.Info)
+
+	dbs, dbName := storetesting.DbOpenTest(t, inMem)
+	storetesting.SetDbLogging(t, dbs)
+
+	err := dbs.Wdb.Atomic(func(ctx context.Context, tx *sql.Tx) error {
+		sqlitedriver.AccountsInitTest(t, tx, make(map[basics.Address]basics.AccountData), protocol.ConsensusCurrentVersion)
+		return nil
+	})
+	require.NoError(t, err)
+
+	defer func() {
+		dbs.Close()
+		require.NoError(t, os.Remove(dbName))
+	}()
+
+	targetVersion := int32(10)
+
+	err = dbs.Wdb.Atomic(func(ctx context.Context, tx *sql.Tx) (err0 error) {
+		_, err0 = tx.ExecContext(ctx, fmt.Sprintf("PRAGMA user_version = %d", targetVersion-1))
+		return
+	})
+	require.NoError(t, err)
+
+	// +---------------------------------------------------------+
+	// | Above is creating a ledger with tracker DB of version 9 |
+	// +---------------------------------------------------------+
```

Comment:
seems like repo standard is to put the comment above the code that is commented upon